docs/references/magic_link_login.md

```diff
@@ -23,6 +23,15 @@ in the **app/Config/Auth.php** file.
 public int $magicLinkLifetime = HOUR;
 ```
 
+### Magic Link Force Login
+
+By default, Magic Link will throw Logic exception if current user session is exist. If you want replace existing user session with new user session change it to `true`
```

Comment:
LogicException means, "your code has a bug, so please fix it".
So I never accept a flag to disable LogicException.
This PR seems to be wrong implementation.

First of all, why a logged in user uses magic link login?

Reply:
I submit a magicLink request using the inconito browser, then the magicLink sends an email to the main browser. when opened it will cause a LogicException because in my main browser I am still logged in.

Reply:
But this scenario rarely happens in real cases. this case often occurs when development involves switching accounts with specific roles.